starport/pkg/cosmosaccount/cosmosaccount.go

```diff
@@ -253,6 +253,26 @@ func (r Registry) GetByName(name string) (Account, error) {
 	return acc, nil
 }
 
+// GetByAddress returns an account by its address.
+func (r Registry) GetByAddress(address string) (Account, error) {
```

Comment:
```suggestion
func (r Registry) GetByAddress(address, accountPrefix string) (Account, error) {
```

cosmosaccount is a generic pkg, it should be able to work with different accountPrefixes and not only cosmos.